src/parsers/manifest/dash/parse_mpd.ts

```diff
@@ -216,7 +221,8 @@ function parseCompleteIntermediateRepresentation(
                           isDynamic,
                           receivedTime: args.manifestReceivedTime,
                           timeShiftBufferDepth,
-                          xlinkInfos };
+                          xlinkInfos,
+                          availabilityTimeOffset };
```

Comment:
You did not follow the alphabetical order like before :p

Reply:
Done

Reply:
No